packages/frontend/src/components/TestSubstep/index.tsx

```diff
@@ -163,11 +165,22 @@ function TestSubstep(props: TestSubstepProps): React.ReactElement {
 
           {stepsWithVariables.length === 1 && (
             <Box sx={{ width: '100%' }}>
-              <Alert severity="info" sx={{ width: '100%' }}>
-                We found these variables:
-              </Alert>
+              <Infobox
+                icon={<BiCheck style={{ color: '#0F796F' }} />}
```

Comment:
nit: can directly use `color` prop.

Let's move this to a chakra token in a later PR.